frontend/src/tree/TreeItemContextMenu.tsx

```diff
@@ -10,16 +10,38 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
+import { useMutation } from '@apollo/client';
+import { IconButton, Snackbar } from '@material-ui/core';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
 import ListItemText from '@material-ui/core/ListItemText';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import CloseIcon from '@material-ui/icons/Close';
 import DeleteIcon from '@material-ui/icons/Delete';
 import EditIcon from '@material-ui/icons/Edit';
-import { TreeItemContextMenuComponentProps } from 'index';
-import React, { useContext } from 'react';
+import gql from 'graphql-tag';
+import {
+  GQLDeleteTreeItemData,
+  GQLDeleteTreeItemInput,
+  GQLDeleteTreeItemVariables,
+  TreeItemContextMenuComponentProps,
+  TreeItemContextMenuState,
+} from 'index';
```

Comment:
Do not import things from "Index", import them from the relevant file. Here `tree/TreeItemContextMenu.types`